provider/resources.go

```diff
@@ -163,6 +163,7 @@ func Provider() tfbridge.ProviderInfo {
 			GenerateResourceContainerTypes: true,
 		},
 		CSharp: &tfbridge.CSharpInfo{
+			RootNamespace: "Pulumi",
```

Comment:
`RootNamespace` shouldn't be `Pulumi`, as this is the namespace Pulumi (the company) uses to publish their packages. You need to pick something Twingate specific.

Reply:
@ringods We can make the RootNamespace `Twingate` . That will make the nuget package `Twingate.Twingate`. What about `PulumiTwingate.Twingate`? What is the standard naming convention you see for companies not using the Pulumi or Pulumiverse as the RootNamespace?